sda-commons-server-kafka/src/test/java/org/sdase/commons/server/kafka/health/KafkaWithoutHealthCheckIT.java

```diff
@@ -0,0 +1,120 @@
+package org.sdase.commons.server.kafka.health;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.salesforce.kafka.test.KafkaBroker;
+import com.salesforce.kafka.test.junit4.SharedKafkaTestResource;
+import io.dropwizard.testing.junit.DropwizardAppRule;
+import java.util.SortedSet;
+import java.util.stream.Collectors;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.rules.RuleChain;
+import org.junit.rules.TestRule;
+import org.sdase.commons.server.kafka.KafkaConfiguration;
+import org.sdase.commons.server.kafka.config.*;
+import org.sdase.commons.server.kafka.dropwizard.KafkaTestConfiguration;
+import org.sdase.commons.server.kafka.dropwizard.KafkaWithoutHealthCheckTestApplication;
+import org.sdase.commons.server.testing.DropwizardRuleHelper;
+import org.sdase.commons.server.testing.LazyRule;
+
+public class KafkaWithoutHealthCheckIT {
+
+  private static final SharedKafkaTestResource KAFKA =
+      new SharedKafkaTestResource()
+          // we only need one consumer offsets partition
+          .withBrokerProperty("offsets.topic.num.partitions", "1")
+          // we don't need to wait that a consumer group rebalances since we always start with a
+          // fresh kafka instance
+          .withBrokerProperty("group.initial.rebalance.delay.ms", "0");
+
+  private static final String CONSUMER = "consumer";
+  private static final String PRODUCER = "producer";
+
+  private static final String LISTENER_CONFIG_ASYNC = "async";
+
+  private static final LazyRule<DropwizardAppRule<KafkaTestConfiguration>> DROPWIZARD_APP_RULE =
+      new LazyRule<>(
+          () ->
+              DropwizardRuleHelper.dropwizardTestAppFrom(
+                      KafkaWithoutHealthCheckTestApplication.class)
+                  .withConfigFrom(KafkaTestConfiguration::new)
+                  .withRandomPorts()
+                  .withConfigurationModifier(
+                      c -> {
+                        KafkaConfiguration kafka = c.getKafka();
+
+                        kafka.setBrokers(
+                            KAFKA.getKafkaBrokers().stream()
```

Comment:
Well this programmatic configuration is pretty long. Probably copied from somewhere else? Don't think it's worth a refactoring.

Reply:
Yes its copied from another test. I already stripped some configurations but I guess it could be reduced even further if thats necessary.

Reply:
Maybe that helps as an example what is really required right now: https://github.com/SDA-SE/partner-consent-stack/blob/feat/events/consent-store-service/src/test/java/com/sdase/service/consent/events/EventManagerIT.java#L40-L49

Reply:
But I would keep the withBrokerProperty, that seems to be some kind of performance optimization for the tests